src/components/Page-Title/Page-Title.css

```diff
@@ -1,3 +1,7 @@
 .-text--h3.-text--boldest.-text--900 {
     font-weight: 900!important;
 }
+
+.chi .chi-button.-icon {
```

Comment:
Why is this necessary?
The right side spacing is misaligned because of this, right? Can we reduce this instead?
<img width="1440" alt="Screenshot 2021-06-24 at 14 58 33" src="https://user-images.githubusercontent.com/25136131/123267454-51419900-d4fd-11eb-8c12-301f2e0c5ded.png">